GIS.md

```diff
@@ -0,0 +1,197 @@
+would like to run the installer in. 
+
+The `oem` skips user creation during installation, meaning that user creation will be handled by Gnome Initial Setup 
+post installation.
+
+The following `whitelable.yaml` placed in `/usr/share/desktop-provision/whitelabel.yaml` on your LiveCD is sufficient 
+to skip user creation during installation, provided you are using the new Ubuntu Desktop Bootstrap installer:
+
+```
+mode: oem
+```
+
+A comprehensive guild to customizing a LiveCD can be found [here](https://help.ubuntu.com/community/LiveCDCustomization).
+
+## Triggering Gnome-Initial-Setup without modifying the LiveCD
+
+Its possible to test the Gnome Initial Setup user creation flow via the Ubuntu Desktop Bootstrap installer without first 
+editing your LiveCD, provided Gnome Initial Setup is already present on your ISO.
+
+### Install ubuntu-desktop-bootstrap
+
+The following command installs the Ubuntu Desktop Bootstrap Flutter installer
+```
+sudo snap install ubuntu-desktop-bootstrap --classic
+```
+
+### Write a whitelabel.yaml to set OEM mode
+
+These commands create a `whitelabel.yaml` in the location Ubuntu Desktop Bootstrap will look for them.
+```
+sudo mkdir -p /usr/share/desktop-provision
+
+sudo bash -c 'echo "mode: oem" > /usr/share/desktop-provision/whitelabel.yaml'
+```
+
+### Launch ubuntu-desktop-bootstrap
+
+Finally, launch the installer. You should be no be prompted for user account creation, which will be handled after the 
+installer finishes
+```
+/snap/bin/ubuntu-desktop-bootstrap --try-or-install
+```
+# Triggering Gnome-Initial-Setup via an autoinstall.yaml
+
+These sections are adapted from the subiquity [documentation](https://canonical-subiquity.readthedocs-hosted.com/en/latest/howto/autoinstall-quickstart.html), updated for triggering Gnome Initial Setup with the Ubuntu Desktop ISO
+
+## Using an autoinstall provided over the network
+
+### Getting an ISO
+You can find a desktop ISO to download on the [official Ubuntu website](https://ubuntu.com/download/desktop). (Currently the latest is Ubuntu 24.04 LTS)
+
+### Mount the ISO
+Once downloaded, you can mount the ISO to make it accessible via from a local directory. Change `<version-number>` in 
+the following command to match the release ISO you downloaded :
+
+```
+sudo mount -r ~/Downloads/ubuntu-<version-number>-amd64.iso /mnt
+```
+
+### Write your autoinstall configuration
+Create your clout-init configuration, making sure to leave the identity section out so that no user is created during 
+installation:
+
+```
+mkdir -p ~/www
+cd ~/www
+cat > user-data << 'EOF'
+#cloud-config
+autoinstall:
+  version: 1
+EOF
+touch meta-data
+```
+
+### Serve the cloud-init configuration over HTTP
+
+```
+cd ~/www
+python3 -m http.server 3003
+```
+
+### Create a target disk
+
+Create the target VM disk for the installation. 25G is the minimum recommended storage allocation for Ubuntu Desktop:
```

Comment:
```suggestion
Create the target VM disk for the installation. 25G is the minimum recommended storage allocation for Ubuntu Desktop:
```

256 what? GB?!

Reply:
Hehe, its a G not a 6, but let me change that to 25GB for clarity :slightly_smiling_face:

Reply:
Omfg, clearly need stronger contacts :laughing: